spec/search.spec.js

```diff
@@ -105,8 +114,8 @@ describe('search', function() {
             animation.add(element('stub'));
             animation.add(searchFor);
 
-            expect(animation.find('/foobar')).toBe(searchFor);
-            expect(animation.findAll('/foobar')[0]).toBe(searchFor);
+            expect(animation.find('//foobar')).toBe(searchFor);
```

Comment:
( [The line](https://github.com/Animatron/player/pull/352/files#diff-450f96d427a7e52e0cc876bee087c640R95) where it happens on import. )